pkg/sql/logictest/testdata/logic_test/enums

```diff
@@ -410,3 +410,42 @@ CREATE TABLE t1 (
    INDEX i (x ASC),
    FAMILY "primary" (x, rowid)
 )
+
+# Test that the implicit array type has been created, and that we can use it.
```

Comment:
Can we test drops too?

Reply:
Can't drop types yet, will have to come back and add that when we do.